pkg/meta/base.go

```diff
@@ -216,7 +216,7 @@ func (m *baseMeta) refreshSession() {
 }
 
 func (m *baseMeta) CleanStaleSessions() {
-	sids, err := m.en.doFindStaleSessions(time.Now().Add(time.Minute*-5).Unix(), 1000)
```

Comment:
There will be a problem when a client use 60s as heartbeat, but another one use 10s, then the second one will remove the first one as stale session.

Reply:
Yes, so users must use the same heartbeat value for all sessions.
A maybe better alternative is to set the expire time rather than the current time stamp, but it's not compatible with old clients.